service/history/workflow/context.go

```diff
@@ -916,13 +916,19 @@ func (c *ContextImpl) ReapplyEvents(
 // TODO: remove `ms` parameter again (added since it's not possible to initialize a new Context with a specific MutableState)
 func (c *ContextImpl) UpdateRegistry(ctx context.Context, ms MutableState) update.Registry {
```

Comment:
Maybe rename the `ms` argument `fallbackMutableState`

Reply:
I hope that `TODO` above will be addressed soon and it will be gone all together.